src/matrix/room/timeline/entries/BaseEventEntry.js

```diff
@@ -155,55 +156,26 @@ export class BaseEventEntry extends BaseEntry {
         return createAnnotation(this.id, key);
     }
 
-    _fallbackBlurb() {
-        switch (this.content.msgtype) {
-            case "m.file":
-                return "sent a file.";
-            case "m.image":
-                return "sent an image.";
-            case "m.video":
-                return "sent a video.";
-            case "m.audio":
-                return "sent an audio file.";
-        }
-    }
-
-    _fallbackPrefix() {
-        return this.content.msgtype === "m.emote" ? "* " : "";
-    }
-
-    get _formattedBody() {
-        return this.content.formatted_body || (this.content.body && htmlEscape(this.content.body));
-    }
-
-    get _plainBody() {
-        return this.content.body;
-    }
-
-    _replyFormattedFallback() {
-        const body = this._fallbackBlurb() || this._formattedBody || "";
-        const prefix = this._fallbackPrefix();
-        return `<mx-reply>
-          <blockquote>
-            In reply to
-            ${prefix}<a href="https://matrix.to/#/${this.sender}">${this.displayName || this.sender}</a>
-            <br />
-            ${body}
-          </blockquote>
-        </mx-reply>`;
-    }
-
-    _replyPlainFallback() {
-        const body = this._fallbackBlurb() || this._plainBody || "";
-        const bodyLines = body.split("\n");
-        bodyLines[0] = `> <${this.sender}> ${bodyLines[0]}`
-        return bodyLines.join("\n> ");
-    }
-
     reply(msgtype, body) {
         // TODO check for absense of sender / body / msgtype / etc?
-        const newBody = this._replyPlainFallback() + '\n\n' + body;
-        const newFormattedBody = this._replyFormattedFallback() + htmlEscape(body);
+        let blurb = fallbackBlurb(this.content.msgtype);
+        const prefix = fallbackPrefix(this.content.msgtype);
+        const sender = this.sender;
+        const name = this.displayName || sender;
```

Comment:
Given that all the properties used on `this` here are public, shall we just put all of this in `reply.js` and pass in the entry in that function? Just to not have HTML stuff in here ...